spec/content_tagger/linking_related_content_spec.rb

```diff
@@ -0,0 +1,47 @@
+feature "Adding related content to Publisher content", new: true, content_tagger: true, frontend: true, publisher: true do
+  include ContentTaggerHelpers
+  include PublisherHelpers
+
+  let(:guide_title) { "Transaction with related content " + SecureRandom.uuid }
+  let(:guide_slug) { "transaction-with-related-content-" + SecureRandom.uuid }
+  let(:related_content_title) { "Related transaction " + SecureRandom.uuid }
+  let(:related_content_slug) { "related-transaction-" + SecureRandom.uuid }
+
+  scenario "Adding a taxon to a publisher guide" do
```

Comment:
should taxon here be related content? (not sure if they're synonyms or not)

Reply:
Good spot! That is incorrect.